initialization/diskstate.go

```diff
@@ -15,14 +15,19 @@ func NewDiskState(datadir string, bitsPerLabel uint) *DiskState {
 }
 
 func (d *DiskState) NumLabelsWritten() (uint64, error) {
-	numBytesWritten, err := d.NumBytesWritten()
+	numBytesWritten, _, err := d.NumBytesWritten()
 	if err != nil {
 		return 0, err
 	}
 
 	return shared.NumLabels(numBytesWritten, d.bitsPerLabel), nil
 }
 
-func (d *DiskState) NumBytesWritten() (uint64, error) {
+func (d *DiskState) NumBytesWritten() (uint64, int, error) {
```

Comment:
`d.NumBytesWritten` imo should not return the number of files as second parameter.

Reply:
Covered in https://github.com/spacemeshos/post/pull/94#discussion_r1045858080.